src/components/DisplayEmbed/DisplayExternal.jsx

```diff
@@ -154,6 +154,10 @@ export class DisplayExternal extends Component {
     if (!allowedProvider) {
       return errorHolder();
     }
+
+    if (!src || !type) {
+      return <div></div>;
```

Comment:
kan man gjøre sånn her?
```suggestion
      return <div/>;
```

Lurer og på om det er noe annet element som gir mer mening å returnere? Sånn UU set så er det oppfordret til å ikke bruke tags som ikke er selvforklarende. Har ikke sett nok på det enda til å komme med noe bedre forslag tho

Reply:
Sikkert andre muligheter ja, men dette er bare den enkleste måten å håndtere at vi vil ungå prøve å bruke src/type uten src/type